src/internal/operators/switchMap.ts

```diff
@@ -36,6 +36,16 @@ export function switchMap<T, R, O extends ObservableInput<any>>(project: (value:
  * subsequent inner Observables.
  *
  * ## Example
+ * Generate new Observable according to source Observable values
+ * ```javascript
```

Comment:
Can you mark the snippet as `ts` instead? This will make it open in the right file format when the "Edit in StackBlitz" button is clicked.

Reply:
resolved in new commit